tests/manifold/spherical_manifold_03.output

```diff
@@ -1,3 +1,3 @@
 
-DEAL::Testing dim 2, spacedim 2
-DEAL::Testing dim 3, spacedim 3
+DEAL::1.57080 0.00000 0.00000
+DEAL::0.00000 1.57080 0.00000
```

Comment:
No, don't squash. What you think may now be unnecessary may still test something orthogonal to other tests and may expose a problem later on. I think that every test you ever wrote should end up in the repository, even if you think that it is not strictly necessary any more.